core/templates/pages/exploration-editor-page/services/exploration-correctness-feedback.service.ts

```diff
@@ -44,12 +44,7 @@ export class ExplorationCorrectnessFeedbackService extends
   }
 
   isEnabled(): string | boolean {
```

Comment:
Why do we need this service at all?

Reply:
Dropped the service as it was unnecessary